### DISCUSSION TITLE
www subdomain keeps showing ERR_CERT_DATE_INVALID

### DISCUSSION THREAD

**Discussion by pkhivesara** [vercel/community#4092] [Help] [answered]
### Summary

I have a domain purchased through Google Domains, using Google's nameservers, hosted on vercel. I have SSL via letsencrypt on my domain (foo.com for example). But for some reason the subdomain (www.foo.com) keeps saying that ERR_CERT_DATE_INVALID.

Looking at vercel, it appears that a valid certificate is in place on foo.com. but for some reason www keeps getting this error.

What am I missing here?

Thanks!

### Steps to Reproduce

Type in www.foo.com to see the error

**Comment by amyegan** [comment, marked as answer]
Hi @pkhivesara. When you added the domain, did you set it to [redirect the `www` subdomain](https://vercel.com/docs/projects/domains/deploying-and-redirecting#redirecting-www-domains) to the apex domain? Or do you have it configured differently?

**Reply by pkhivesara** [reply]
Never mind, I got it working. I had to flip over the redirects to point from www to non-www. Thanks!

**Comment by pkhivesara** [comment]
Hi @amyegan Thanks for the reply! I added the www subdomain as per your suggestion, which helped. However, it now seems to fail to redirect.

![Screen Shot 2023-09-15 at 1 05 51 PM](https://github.com/vercel/community/assets/6237488/25423207-0f02-41e8-9777-77fa08e6ab25)
